packages/data-models/src/definition-set.js

```diff
@@ -1,25 +1,56 @@
-import Definition from './definition'
+import Definition from './definition.js'
+import Language from './language.js'
 import LMF from './language_model_factory.js'
 
 export default class DefinitionSet {
-  constructor (lemmaWord, languageID) {
+  /**
+   * @param {string} lemmaWord - A word for which a definition set was created.
+   * @param {Language} language - A language on what a text of a definitions in a definition set is written.
```

Comment:
I believe the language property on the definition set is the language of the lemma and not the definition text itself.

Reply:
Oh yes, that's correct! Fixed the description.